### PR TITLE
Add express structure and config

### PR DESCRIPTION
**Pull request by Emmanuel262**
## What does this PR do?

- It add express structure especially folders.
- It provide a welcome endpoint to test setup
- Setup API documentation with Swagger
- It provide test of an endpoint provided above.

## Description of Task to be completed?

- Creating folders for each tasks for improving code maintainability and reusability.
- Provide a welcome message to the API consumers before they start consumption.
- Setup API documentation which guides API consumers how to consume an API and guidelines for starting.
- Each task and function write to complete task needs to be tested for good practice of an API.

## How should this be manually tested?

**Clone the repository, the following command to install all depandencies.**
```
    npm install
```
or 
```
    npm i
```

- start the server with `npm start`
- Go to `http://localhost:3000/` to see welcome message
- Go to `http://localhost:3000/api-docs` to see API documentation of an api.

## Testing tools to use.

---

- [x] postman

### CODE REVIEW

**Review by cryptotearer — COMMENTED** [review]
Incredible work brother!

**Review by malaba6 — CHANGES_REQUESTED** [review]
Good job @Emmanuel262. A few things to point out though.
- First, I don't see the folders, models and controllers
- Second, you need to remove the modules folder. What it it for? You can move that file in there to models or controlers and remove it later since it's just a test file.
- third you need to put all the sequelize folders inside one folder that you can either call `db` or `sequelize` for maintainability purpose.
- Fourth, why do you have both `app.js` and `server.js`? please remove one of them, and I am thinking it `app.js`

File: src/server.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  },
+  apis: ["./src/routes/*.js"],
+};
+
+const specs = swaggerJSDoc(options);
+const app = express();
+
+app.use("/api-docs", swaggerUI.serve, swaggerUI.setup(specs));
+
+app.get("/junior", (req, res) => {
```
Comment (malaba6) [inline src/server.js:40]: What is this endpoint for? Please remove it if it has nothing to do with the product

File: test/home_route.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const chai = require("chai");
```
Comment (malaba6) [inline test/home_route.js]: please write es6 not es5 code here

**Review by malaba6 — CHANGES_REQUESTED** [review]
There seems to be conflict with develop on this branch.
Besides github actions are failing, please make sure to fix that

File: test/addition.js [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
-    });
-});
\ No newline at end of file
+const expect = _expect;
+
+describe(`addition`, () => {
+  it(`it should add 2 number`, (done) => {
+    add(1, 4)
+      .then((result) => {
+        console.log(result);
```
Comment (Anguandia) [inline test/addition.js]: Please remove all console statments

File: test/home_routes.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+// import chai from "chai";
+// import chaiHttp from "chai-http";
+// import app from "../src/server.js";
+
+// import { chai, chaiHttp, app } from "./setup.test";
```
Comment (Anguandia) [inline test/home_routes.js]: Please remove code you do not need

File: test/setup.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import app from "../src/server.js";
+import should from "chai";
+
+chai.should();
+chai.use(chaiHttp);
+const { expect } = chai;
+
+// export { chai, chaiHttp, app, expect };
```
Comment (Anguandia) [inline test/setup.js]: Same here

**Review by Anguandia — CHANGES_REQUESTED** [review]

**Pull request merged**